README.md

```diff
@@ -1,2 +1,4 @@
-# Demo Project
-This project is just for demo purpose. Change the `README.md` file and commit a pull request and a robot will answer.
+Here should lint error
```

Comment:
This is an in line comment (at `Line 1`) with some **fancy** `markdown` and embedded xml (code highlight)
```xml
  <some>
    <embedded x="ml"/>
    Crazy stuff
  </some>
```